registry/registry.go

```diff
@@ -24,6 +25,9 @@ var (
 	// ErrAlreadyExists is an error returned if an image being pushed
 	// already exists on the remote side
 	ErrAlreadyExists = errors.New("Image already exists")
+	// SessionID is a UUID that identifies the current execution context
+	// and is added to the docker headers in all requests against a registry
+	SessionID = uuid.Generate().String()
```

Comment:
@DavidObando Does this correlate across requests and for a pull will all requests correlate using the same session id?
For e.g. once the manifest request is done, will all the layer pulls have the same session id. 
Also does this Id bleed across requests?